api/coreservice.go

```diff
@@ -1731,3 +1733,45 @@ func (core *coreService) Track(ctx context.Context, start time.Time, method stri
 		Success:      success,
 	}, size)
 }
+
+func (core *coreService) traceTx(ctx context.Context, txctx *tracers.Context, config *tracers.TraceConfig, simulateFn func(ctx context.Context) ([]byte, *action.Receipt, error)) ([]byte, *action.Receipt, any, error) {
```

Comment:
create a private func for tracer initialization instead of tracing with simulateFn?

Reply:
> create a private func for context initialization instead of traceTx with simulateFn?

traceTx (js tracing) has timeout settings, need to put simulateFn into tractTx.

Reply:
it seems no need to put `simulateFn` as param of `traceTx`, can just call `core.SimulateExecution`

Reply:
> it seems no need to put `simulateFn` as param of `traceTx`, can just call `core.SimulateExecution`

If do this, need to pass some `core.SimulateExecution` required arguments